mesmerize_napari/tests/test_core.py

```diff
@@ -118,8 +118,10 @@ def test_mcorr():
     except:
         pytest.fail("Something wrong with setting UUID for batch items")
 
-    assert os.path.join(vid_dir,df.iloc[-1]['input_movie_path']) == \
-           os.path.join(vid_dir, f'{algo}.tif')
+    assert get_full_data_path(df.iloc[-1]['input_movie_path']) == \
+                  Path(vid_dir, f"{algo}.tif") == \
```

Comment:
no this won't work, change the stuff in line 19-20 to `Path` objects and then `Path.joinpath()`

https://docs.python.org/3/library/pathlib.html